tests/shell/shell_recorder/kdb_ls.esr

```diff
@@ -0,0 +1,19 @@
+Storage: dump
+Mountpoint: system/tmount/kdb_ls
+DiffType: File
+
+< kdb mount $File $Mountpoint $Storage
+
+< kdb set $Mountpoint/test bla
```

Comment:
You can and should use the tutorial style syntax or did you have any problems with it?